Update action-groups.md (#123162)
Azure/azure-content: articles/azure-monitor/alerts/action-groups.md

   |Action type|Details  |
   |---------|---------|

>    |Automation Runbook|Use Automation Runbook to automate tasks based on metrics. For example, shutting down resources once certain threshold in the associated budget is met. For information about limits on Automation runbook payloads, see [Automation limits](../../azure-resource-manager/management/azure-subscription-service-limits.md#automation-limits). |

Reviewer: ```suggestion
   |Automation Runbook|Use Automation Runbook to automate tasks based on metrics. For example, shut down resources when a certain threshold in the associated budget is met. For information about limits on Automation runbook payloads, see [Automation limits](../../azure-resource-manager/management/azure-subscription-service-limits.md#automation-limits). |
```
